### PR TITLE
style: gonomics spring cleanup refactor edition eau

### PR DESCRIPTION
**Pull request by edotau**
# Description
- Continuing to clean up files as I add a few more configurations
- added spell check to lint configs
### Checklist before requesting a review

- [x] I performed a self-review of my code
- [x] If it this a core feature, I have added thorough tests
- [x] The command `go fmt` was used on all files included

### Testing

None

### CODE REVIEW

**Review by dasnellings — APPROVED** [review]

File: cmd/bedpeOverlap/bedpeOverlap.go [hunk 1]
```diff
@@ -262,7 +262,7 @@ func main() {
 		log.Fatalf("Error: overlapThreshold must be used with bedSelect")
 	}
 
-	if *overlapBoth == true && *bedSelect == false {
+	if *overlapBoth {
```
Comment (rimangan) [inline cmd/bedpeOverlap/bedpeOverlap.go:265]:
```suggestion
	if *overlapBoth && !*bedSelect {
```

Comment (rimangan) [reply cmd/bedpeOverlap/bedpeOverlap.go:265]: Is this right? We don't want to fatal if overlapBoth is true and bedSelect is also true.

Comment (edotau) [reply cmd/bedpeOverlap/bedpeOverlap.go:265]: Ah nice catch, I misinterpreted the cases, these are positional flags by the user

File: cmd/bedpeOverlap/bedpeOverlap.go [hunk 2]
```diff
@@ -257,12 +261,12 @@ func main() {
...
 
-	//should no longer be needed when overlapThresholdBedPe is implemented
-	if *overlapThreshold != 0 && *bedSelect == false {
+	// should no longer be needed when overlapThresholdBedPe is implemented
+	if *overlapThreshold != 0 && *bedSelect {
 		log.Fatalf("Error: overlapThreshold must be used with bedSelect")
 	}
 
-	if *overlapBoth == true && *bedSelect == false {
+	if *overlapBoth && *bedSelect {
```
Comment (rimangan) [inline cmd/bedpeOverlap/bedpeOverlap.go:269]: So I think this is going to fatal when both are true, but the fatal condition is overlapBoth==true, bedSelect==false.

Comment (edotau) [reply cmd/bedpeOverlap/bedpeOverlap.go]: my bad meant to put `!`

**Review by RavenYL — APPROVED** [review]
This looks good to me! Noting that we can write "return bool check" instead of "if bool check, return true"

**Review by dasnellings — APPROVED** [review]

**Review by rimangan — APPROVED** [review]

**Pull request merged**